src/redux/sagas/utils/arbitraryTxs.ts

```diff
@@ -0,0 +1,80 @@
+import { Interface } from 'ethers/lib/utils';
+
+import { type AddTransactionTableModel } from '~v5/common/ActionSidebar/partials/forms/ArbitraryTxsForm/types.ts';
+
+/**
+ * Parses a string representation of an array into an array of values.
+ * It supports nested arrays and handles strings within arrays.
+ */
+const getArrayFromString = (value: string, type: string) => {
+  if (value === '[]') {
+    return [];
+  }
+
+  if (!value.startsWith('[') || !value.endsWith(']')) {
+    throw new Error('Invalid array format');
```

Comment:
I would propose moving this to translation.

Reply:
Thanks for the suggestion! I had a look throughout the codebase and could not see a pattern of using translations for thrown errors. I also think of this error as a last resort, invalid arrays should have been validated in the UI before getting to this util.